### DISCUSSION TITLE
Why is v3 using Solmate instead of OZ Contracts?

### DISCUSSION THREAD

**Discussion by Koowah** [tinchoabbate/damn-vulnerable-defi#25] [Help / Q&A / Troubleshooting] [answered]
Hi all (@tinchoabbate),

I'm interested in the reasons behind replacing some of `OpenZeppelin`'s utility contracts by `Solmate/Solady`'s in the V3. 
Do they better suit the purpose ? Is it because they are simpler ?

Thanks 😄

**Comment by tinchoabbate** [comment, marked as answer]
I wanted to expose players to a broader set of well-known libraries being used in the wild. So that when you really go into bug hunting, contests or audits, you'll be already familiar not only with OZ building blocks, but also solmate, solady, and whatever other lib I might include in future releases.

**Reply by Koowah** [reply]
@tinchoabbate 

That's an excellent idea. I definetly wouldn't have strayed from OZ contracts if not for these challenges. Very much appreciated !
